doc/modules/curation.rst

```diff
@@ -24,21 +24,21 @@ The merging and splitting operations are handled by the :py:class:`~spikeinterfa
 
     from spikeinterface.curation import CurationSorting
 
-    sorting = run_sorter('kilosort2', recording)
+    sorting = run_sorter(sorter_name='kilosort2', recording=recording)
 
-    cs = CurationSorting(sorting)
+    cs = CurationSorting(parent_sorting=sorting)
```

Comment:
Is there a reason for this? Why do some functions use `parent_sorting` or `parent_recording` and most use `sorting` or `recording`.

Reply:
Yeah, I think the path of least resistance and the most obvious for the user is just to use `recording` and `sorting`. I don't think I would guess that I have a `parent_sorting` vs `child_sorting` at the api level for most functions. (Also less typing).

Reply:
Agreed. It's also a good time to make the switch. Do you think we should deprecate the old arguments or just break compatibility towards the next release?

Reply:
Based on the recent issues with functions and arguments (#1907, #1899, #1678) it might be a better idea to deprecate for at least one release (although tearing off the band-aid would be cleaner like in #1865, which I would prefer), it seems like that change caused a bit of confusion for users.

Reply:
well this is only changing an argument name that it's probably not really used by most users. I think that most users will use `banpass_filter(recording)` and not `bandpass_filter(parent_recording=recording)` (same for `channel_slice`, other preprocessors, etc..). Keeping back-compatibility would mean adding a temporary argument and setting the main `recording` to None, which would possibly require to make some other args None by default...

Let's just make the switch. I checked and it's not that many files :)